docs/command_line_tool/phylum_package.md

```diff
@@ -7,7 +7,7 @@ hidden: false
 Retrieve the details of a specific package
 
 ```sh
-Usage: phylum package [OPTIONS] <name> <version>
+Usage: phylum package [OPTIONS] --package-type <type> <name> <version>
```

Comment:
Was `--package-type` intentionally left as an option instead of an argument? The expectation was that the signature would look more like:

```
Usage: phylum package [OPTIONS] <name> <version> <type>
```

It seems awkward to have a required option.

Reply:
I kept it as an argument to not unnecessarily break stuff, but we could change it. The original issue didn't suggest doing so either so I just went with what is "safe".

Reply:
Roger that...and understood. In this case, the assumption is that there are no consumers of this command to break...but maybe @furi0us333 has more insight there.

Reply:
Impact should be minimal and I think if it is a mandatory field, it isn't an "option" anymore. Seems like argument makes the most sense to me. Sorry, I didn't mean my choice of words in the issue to be loaded.